projects/module-1/which-beyonce-group.md

```diff
@@ -0,0 +1,220 @@
+* **1: Very behind** - student needs intervention to get back on track
+
+## Functionality
+
+* **4:** Iterations **0-6 are fully functional** with no bugs, and at least one extension has been fully implemented without bugs.
+* **3:** Iterations **0-6 are fully functional**. One small bug may exist. (Point of clarity - a small bug is not a missing feature or a completely broken feature; it's a feature that usually works but behaves unexpectedly in certain edge cases.)
+* **2:** Iterations **0-5 are fully functional** with no bugs.
+* **1:** Iterations 0-4 are functional.
+
+## Comp Recreation (UI and UX)
+
+The topic does not have to be Beyonce. Besides assets and name of app, however, students are expected to recreate the comp.
+
+* **4:** Additional elements that have been added **match** the visuals established in the comps. **Finished touches** such as `cursor: pointer` and additional visual cues on hover are thoughtfully implemented.
+* **3:** Application implements all major comp details accurately and correctly on desktop and large TV. **Additional elements added generally match** the visuals established in the comps, but may be slightly awkward. **Nothing about the UI is confusing** to a user who has never seen this application before.
+* **2:** Application implements all major comp details accurately and correctly on desktop. Some UI may be slightly confusing or unclear.
+* **1:** Application implements most major comp details accurately and correctly on desktop.
+
+## HTML
+
+* **4:** Application uses BEM, SMACCS or another set of **naming conventions** for classes; all developers can explain those naming conventions. The form uses **WAI-ARIA** and is fully accessible.
+* **3:** Application uses markup that is **easy to read** and follows across **naming conventions**.
+* **2:** Application uses markup that incorporates **semantic HTML** elements and has a simple, clean HTML structure.
+* **1:** Application uses markup that meets the criteria of the [Turing HTML Style Guide](https://github.com/turingschool-examples/html).
+
+## JavaScript
+
+* **4:** **Code is well refactored** and demonstrates developer empathy. No global variables are used aside from query selectors, start time, and instances of `Player` and `Deck`. If you feel you need more because you are building out additional functionality that requires a global variable, please check in with an instructor. All functions are **less than 10 lines long** or otherwise demonstrate SRP.
+* **3:** The application correctly **implements a data model** for the `Card` and `Deck` classes, including all required methods. Arguments and parameters are used to limit global variables. **Function declarations** are used over anonymous functions in event listeners. Developers can speak to the **role of each class**. The separation of data model logic and presentational logic is clear and can be explained by all developers in the team.
```

Comment:
To be even more clear, I think we might want to add "All DOM manipulation is handled in `main.js`" because they reallllly like to put it in the class methods.